<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="readMsstyle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>read msstyle</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="msstylePath">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>/path/to/aero.msstyle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>push button info:</string>
    </property>
   </widget>
   <widget class="QLabel" name="classname">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>class name here</string>
    </property>
   </widget>
   <widget class="QLabel" name="partid">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>part id here</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateid">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>normal state id here</string>
    </property>
   </widget>
   <widget class="QLabel" name="extractedPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>msstyle extraction path here</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagefile">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>texture here</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>msstyle version here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
